src/hooks/useStore.js

```diff
@@ -2,10 +2,10 @@ import create from 'zustand';
 
 const useStore = create(set => ({
 	cartProducts: [],
-	addToCart: name => {
+	addToCart: id => {
 		set(state => {
 			return {
-				cartProducts: [...state.cartProducts, name],
+				cartProducts: [...state.cartProducts, id],
```

Comment:
The naming of `cardProducts` is confusing. Is it correct you just saving `id`s in it? I would have assumed that ? cardProducts` holds some kind of product information?

Maybe renaming `cartProducts` to a better name, that shows, what is holds or I found a bug? ;-)